.codesandbox/ci.json

```diff
@@ -1,5 +1,6 @@
 {
-  "installCommand": "bootstrap",
+  "installCommand": "codesandbox",
```

Comment:
Can we just run `build`? The `build-no-bundle` run `babel` on every package while `build` will rollup `babel-parsers`.

Reply:
The problem is that `build` was too slow and froze codesandbox

Reply:
yeah we purposely didn't because we don't need it in this case, and it was too slow for some reason: https://ci.codesandbox.io/status/hzoo/babel/pr/1

Reply:
I guess it comes from building `babel-standalone`.

An ugly workaround is `BABEL_COVERAGE=true make build`, which will skip building standalone.